src/Commands/RunCommand.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: nitishkumar
+ * Date: 3/8/18
+ * Time: 9:51 AM
+ */
+
+namespace Hyn\Tenancy\Commands;
+
+use Hyn\Tenancy\Contracts\Repositories\WebsiteRepository;
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Console\Kernel;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Hyn\Tenancy\Models\Website;
+
+class RunCommand extends Command
+{
+    protected $signature = 'tenancy:run {tenantcommand}
+        {--tenant= : The tenant(s) to apply on; use {all|5,8}}
+        {--arguments= : Arguments for the delegated command} 
+        {--options= : Options to pass on to the delegated command}
+    ';
+
+    protected $description = 'Run another artisan command in a tenant configuration';
+
+    public function fire()
+    {
+        $websites = $this->getWebsitesFromOption();
+        $newArgv = ['artisan', $this->argument('tenantcommand')];
+        if ($arguments = $this->option('arguments')) {
```

Comment:
Can you explain this a little bit please?

Reply:
Instead of a single parameter on this command `--arguments`, you can just read the arguments as a collection with `$this->arguments()`.  This will return your `{command}` argument in the set, so you'll need to filter that value out.

After doing this, the command could be called like:
`artisan tenancy:run command arg1 arg2`
instead of
`artisan tenancy:run command --arguments=arg1,arg2`

Reply:
And how the filtration will go?

Reply:
That's really up to you I guess, but you could do something like:
```
$ignoredArgs = [$command, /* others */];
array_filter($this->arguments(), function($arg) use ($ignoredArgs) {
    return !in_array($arg, $ignoredArgs);
});
```